### DISCUSSION TITLE
Diver [gcs] is not supported

### DISCUSSION THREAD

**Discussion by mobreviews** [spatie/laravel-google-cloud-storage#48] [Q&A] [answered]
I've done the composer install and added the section to my filesystem.php and configured my env variables but when I try to upload to it I get a message -
```
message: "Driver [gcs] is not supported.", exception: "InvalidArgumentException",…}
exception : "InvalidArgumentException"
file : "/app/vendor/laravel/framework/src/Illuminate/Filesystem/FilesystemManager.php"
line :  148
message :  "Driver [gcs] is not supported."
```
Ay ideas?  I've only set the following 2 env variables as it appears all the others are optional.
```
GOOGLE_CLOUD_STORAGE_BUCKET=laravel_storage
GOOGLE_CLOUD_KEY_FILE=/app/360204-c02cb6578b45.json
```
before you say it my entire Laravel app is in an app folder so this is not the normal Laravel /app folder that folder is found at /app/app.

**Comment by mobreviews** [comment, marked as answer]
Please ignore this.  I did a composer update and it appears to have solved the issue.  There must have been an old dependency in my lock file somewhere.

**Comment by AnoNate** [comment]
I fixed this by running
`composer dump-autoload`
Hope it might help someone.
